root/statistics/stats.js

```diff
@@ -164,6 +164,132 @@ const stats = {
     description: l('Count of all CDStub tracks'),
     label: l('CDStub tracks'),
   },
+  'count.collection': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Count of all collections'),
+    label: l('Collections'),
+  },
+  'count.collection.has_collaborators': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Count of all collections with one or more collaborators'),
+    label: l('Collections with collaborators'),
+  },
+  'count.collection.private': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Count of all private collections'),
+    label: l('Private collections'),
+  },
+  'count.collection.public': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Count of all public collections'),
+    label: l('Public collections'),
+  },
+  'count.collection.type.area': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Collections of areas'),
+    label: l('Area collections'),
+  },
+  'count.collection.type.artist': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Collections of artists'),
+    label: l('Artist collections'),
+  },
+  'count.collection.type.attending': {
+    category: 'collection',
+    color: '#ff0000',
+    description: l('Collections with type set to Attending'),
+    label: l('Collections of type Attending'),
```

Comment:
These are saying the exact same thing with slightly different wording, and I'm not sure why one wording is considered a description and another is a label. :D  Maybe we should leave description blank/null if there's nothing meaningful to add beyond the label?  Even "Artist collections" vs. "Collections of artists," and probably some others feel the same way, i.e. they're minimal re-wordings that don't feel useful.  Though I'm sure we already have existing items that do this.

Reply:
Yes, a ton of these seem to be "label is just shorter to fit on the sidebar better", but with stuff like this where it's hard to make a proper short label, it starts to lose the point a bit...

Reply:
Why the label isn’t just “of type Attending”? Like for “Artists” “of type Person” in [Overview](https://musicbrainz.org/statistics).

Reply:
The label is (IIRC) for the timeline sidebar, so it needs to be complete.

Reply:
Most of the current descriptions are useless (e.g. just adding “Count of” ahead of the label text), but some labels are not clear enough. They should probably be replaced with the corresponding descriptions if having long labels doesn’t cause display issues.
For example:
* “Artists not in ACs” isn’t telling as much as “Artists with no appearances in artist credits” (and even then “artist credits” should probably link to the documentation),
* “Unknown-gender Artists” is clarified with the description “Artists with gender unset (non-group artists)”,
* “Failed edits (prerequisite)” isn’t as clear as “All edits that have failed prerequisite checks” either.

I didn’t even check all the 130+ descriptions. It seems that “description” was expected to be self-explanatory while “label” depends on the grouped display. Some git archeology might help with reminding its expected purpose.

Would you be alright with moving this change into a new PR so the current one can be merged sooner?